### ISSUE TITLE
Expose sorting operator in DataObject

### ISSUE THREAD

**Issue by GuillemBarroso** [pyansys/pydpf-post#220]
### Description of the feature

The ``DataObject`` class contains data for certain field/s. The user may want to select some time-steps and/or nodes as

```py
disp = simulation.displacement(nodes=[4,6], time=[-1], component='x')
``` 
Some users may require the data in ``disp`` to be sorted by node ID. To do so, the user will have two different ways to sort the ``DataObject``. Using ``.sort()`` method similar to ``numpy``, see [here](https://numpy.org/doc/stable/reference/generated/numpy.sort.html):

```py
array = disp.sort().as_array()
``` 

or by adding an extra argument in the ``.as_array()`` method:

```py
array = disp.as_array(sort=True)
``` 

**Questions**
- Do we need both implementations (``.sort()`` method in ``DataObject`` and optional argument in ``.as_array()`` and ``.as_dataframe``)?
- Do we only want to sort by node IDs? Do we have more use cases where some users may want to sort using a different criterion?

### Steps for implementing the feature

The sorting will be done in ``data_object.py``.

Use the ``rescope`` scoping operator in the background to sort the data server-side.

### Useful links and references

N/A

**Comment by PProfizi** [comment]
@cbellot000 @rlagha following our discussions, is this the Issue you mentioned as not required anymore concerning sorting?

**Comment by cbellot000** [comment]
@PProfizi In my opinion, it's still a good option to have (but the priority is not very high relying on customer feedback).
I updated the issue with:

"""
2 Operators can be used for this purpose:
 - Use the ``rescope`` scoping operator in the background to reorder the data server-side following a scoping order.
 - Use the ``ascending_sort`` or ``descending_sort`` to sort the data by ascending or descending scoping ids.`

"""

**Comment by GuillemBarroso** [comment]
@cbellot000, @PProfizi, yes I agree this may still be relevant but definately not prioritary.
